src/main/java/Models/User/UserAccountDAO.java

```diff
@@ -161,5 +163,36 @@ public static boolean updateUserAccount(UserAccount userAccount) {
         }
     }
 
-}
+    public static void deleteUserAccount(int ID) {
+        Connection con = Database.getInstance().getConnection();
+        try {
+
+            PreparedStatement ps = con.prepareStatement("DELETE FROM USERS WHERE ID = ?");
+            ps.setInt(1, ID);
+            ps.executeUpdate();
+        } catch (SQLException ex) {
+            Logger.getLogger(UserAccountDAO.class.getName()).log(Level.SEVERE, null, ex);
+        }
+    }
+
+    public static void updateUserAccountDetails(int ID, String email, String role, String firstname, String surname) throws SQLException, DuplicateEmailPresentException {
```

Comment:
Does `updateUserAccountDetails` have different functionality to `updateUserAccount` on line 138?

Reply:
updateUserAccountDetails is used to update user details in admin page excluding password
Update user removes user such as doctor, admin and nurse from pending list